src/main/java/org/c4marathon/assignment/board/service/BoardService.java

```diff
@@ -0,0 +1,76 @@
+package org.c4marathon.assignment.board.service;
+
+import lombok.RequiredArgsConstructor;
+import org.c4marathon.assignment.board.domain.Board;
+import org.c4marathon.assignment.board.domain.service.BoardGetService;
+import org.c4marathon.assignment.board.domain.service.BoardSaveService;
+import org.c4marathon.assignment.board.dto.BoardCreateRequest;
+import org.c4marathon.assignment.board.dto.BoardGetAllResponse;
+import org.c4marathon.assignment.board.dto.BoardGetOneResponse;
+import org.c4marathon.assignment.user.domain.User;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.Pageable;
+import org.springframework.security.crypto.bcrypt.BCryptPasswordEncoder;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+@Service
+@RequiredArgsConstructor
+public class BoardService {
+
+    private final BoardSaveService boardSaveService;
+    private final BoardGetService boardGetService;
+    private final BCryptPasswordEncoder encoder;
```

Comment:
```suggestion
    private final PasswordEncoder encoder;
```
구체적인 클래스보다는 상위 인터페이스에 의존하는 것이 좋겠습니다.